project/Dependencies.scala

```diff
@@ -2,7 +2,7 @@ import sbt._
 
 object Versions {
   val circe = "0.11.1"
-  val coursier = "2.0.0-RC3-4"
+  val coursier = "2.0.0-RC4"
```

Comment:
I assume that `RC4` includes https://github.com/coursier/coursier/pull/1403 so will fix https://github.com/fthomas/scala-steward/issues/923.

Reply:
Many thanks for the fix in Coursier, @exoego! The tests here verified that your fix is included in that version.